addons/account/models/account_move.py

```diff
@@ -59,6 +59,13 @@
 
 EMPTY = object()
 
+ALLOWED_MIMETYPES = {
+    'text/csv',
+    'application/vnd.ms-excel',
+    'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+    'application/vnd.oasis.opendocument.spreadsheet',
+}
+
```

Comment:
Why not importing this one to avoid having to maintain two dicts ? :)

https://github.com/odoo/odoo/blob/f63ce0f6d46d28851e3c90dd25164d3122745840/addons/base_import/models/base_import.py#L64-L74